### PR TITLE
[DATA-363] Add integration tests for orbslam

### PR DESCRIPTION
**Pull request by tessavitabile**
This PR adds integration tests for orbslam running in online and offline mode, as well as using a saved map. Once this is merged, it will start running on changes to the slam repo. Also, now that app images are being built for orbslam, I'll make the github actions for rdk grab them so that this integration test can run.

### CODE REVIEW

File: services/slam/builtin/orbslam_int_test.go [hunk 1]
```diff
@@ -56,43 +103,162 @@ func TestOrbslamIntegrationExample(t *testing.T) {
...
-		position.Pose().Orientation().AxisAngles().RZ,
-		position.Pose().Orientation().AxisAngles().Theta)
-	actualMIME, _, pointcloud, err := svc.GetMap(context.Background(), "test", "pointcloud/pcd", nil, false)
+	testPositionAndMap(t, svc)
+
+	// Wait for the final map to be saved
+	for {
+		line, err := logReader.ReadString('\n')
+		test.That(t, err, test.ShouldBeNil)
+		if strings.Contains(line, "Finished saving final map") {
```
Comment (tessavitabile) [inline services/slam/builtin/orbslam_int_test.go:192]: I'll need to add this log line to orbslam.

File: services/slam/builtin/orbslam_int_test.go [hunk 2]
```diff
@@ -56,43 +103,162 @@ func TestOrbslamIntegrationExample(t *testing.T) {
...
+
+	// Make sure we initialize from a saved map
+	logReader = svc.(internal.Service).GetSLAMProcessBufferedLogReader()
+	for {
+		line, err := logReader.ReadString('\n')
+		test.That(t, err, test.ShouldBeNil)
+		if strings.Contains(line, "Initialization of Atlas from file") {
+			break
+		}
+		test.That(t, strings.Contains(line, "Initialization of Atlas from scratch"), test.ShouldBeFalse)
```
Comment (tessavitabile) [inline services/slam/builtin/orbslam_int_test.go:239]: It's possible for this to fail due to https://viam.atlassian.net/browse/DATA-467. If we see that happen in CI, we may need to fix the bug sooner.

**Review by biotinker — APPROVED** [review]
LGTM!

**Review by JohnN193 — APPROVED** [review]
LGTM

**Review by kkufieta — APPROVED** [review]
LGTM

**Pull request merged**